src/components/ChatLog.jsx

```diff
@@ -0,0 +1,45 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+
+const ChatLog = ({entries, handleLike, currentUser}) => {
+    const getChatLog = (entries) => {
+    return entries.map((entry) => {
+    return (
+        <ChatEntry                                 
+            key={entry.id}
+            id={entry.id}
+            sender={entry.sender}
+            body={entry.body}
+            timeStamp={entry.timeStamp}
+            liked={entry.liked}
+            handleLike={handleLike}
+            currentUser={currentUser}
+        />
+    )
+})
+};
+
+return (
+        <section>
+            
+            <ul>{getChatLog(entries)}</ul> */
+        </section>
+    );
+};
```

Comment:
Linter calls out that trailing spaces are not allowed, remove spaces on line 26.
Indentation is off on lines 24 and 29.
`*/` probably from a leftover comment was left in and should be removed on line 27 
```suggestion
  return (
        <section>
            <ul>{getChatLog(entries)}</ul>
        </section>
  );
};
```